directory/src/components/Directory/List.js

```diff
@@ -1029,4 +1029,12 @@ export const list = [
     description:
       "Ramblingsales.com sells crypto apparel and uses BTCPay Server on checkout."
   },
+  {
+    name: "Pedro Gonçalves",
```

Comment:
I think Pedro Goncalves is the owner's name. Maybe name here should be Pirate Hash?

Reply:
Yeah, that's the way submission was made so didn't want to alter. I believe people are used to email forms and associate name with a name. How about we put Project/Company name as the name of the field.

Reply:
Good idea. Will do.

Reply:
I will now modify the store name :)

Reply:
@joerlop fixed now.